lib/msal-angular/package.json

```diff
@@ -60,17 +60,17 @@
     "karma-coverage-istanbul-reporter": "~3.0.2",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
-    "ng-packagr": "^11.2.4",
+    "ng-packagr": "^13.2.1",
     "protractor": "~7.0.0",
-    "rxjs": "~6.5.5",
+    "rxjs": "~7.5.2",
     "ts-node": "~8.3.0",
     "tslib": "^2.0.0",
-    "typescript": "~4.1.5",
-    "zone.js": "~0.10.3"
+    "typescript": "~4.5.5",
+    "zone.js": "~0.11.4"
   },
   "peerDependencies": {
     "@azure/msal-browser": "^2.21.0",
-    "rxjs": "^6.0.0"
+    "rxjs": "^7.5.0"
```

Comment:
Isn't it possible to support both rxjs 6 and 7?

e.g.
```
"rxjs": "^6.5.3 || ^7.5.0",
```

Reply:
Agree, given dropping support for v6 would be a breaking change, which we will not accept.

Reply:
I updated the rxjs range accordingly to support latest 6.x to 7.x.

Reply:
@sandrooco Apologies for the delayed response, thanks for addressing. To confirm, the updated import statements are compatible with v6 and v7 (this was my biggest concern)?

Reply:
@jasonnutter Thanks for checking in.
Actually you are right - the new imports are not compatible. (7.1.0 is the last version with the old imports.) If I'm not mistaken msal can live with 7.x and the customer project with 6.x.

Also I honestly don't understand why dropping 6.x support would be a problem. In my opinion, people should upgrade to the latest 7.x version because 7.5 also is the default when setting up Angular 13 projects.
What do you think?

Reply:
How would this work for older versions of Angular (MSAL Angular v2 still supports Angular 9-12)?

Reply:
i.e. apps that still have rxjs@6 (the point being we can't update to 7 if it would break apps on older versions of rxjs).